packages/agents-hosting/src/app/agentApplication.ts

```diff
@@ -255,7 +261,7 @@ export class AgentApplication<TState extends TurnState> {
   ): this {
     (Array.isArray(type) ? type : [type]).forEach((t) => {
       const selector = this.createActivitySelector(t)
-      this.addRoute(selector, handler, false, authHandlers)
+      this.addRoute(selector, handler, false, RouteRank.Unspecified, authHandlers)
```

Comment:
how a user will set the rank? guessing we should add an optional param to this method, and pass it to addRoute